### DISCUSSION TITLE
关于论文页眉内容设置的问题

### DISCUSSION THREAD

**Discussion by senju666** [sjtug/SJTUThesis#760] [提问]
请问在硕士论文的模板中，在哪里更改左页眉和右页眉的内容
![image](https://user-images.githubusercontent.com/98955078/152339944-181860ef-9c65-41b6-86da-f95367acc4b4.png)
比如想要更改上图中页眉的内容为其他的需要怎么更改，谢谢！

**Comment by skyzh** [comment]
如果是想要更改成左右页不同，可以给 documentclass 加上 twoside,openright 选项。

如果是想更改成其他文本，可以直接搜索 cls 文件中的 `fancyhead` 部分。

**Reply by zhangyu233** [reply]
Hi, 我有和这个同学一样的问题，但是我在sjtuthesis.cls文件中没有找到fancyhead，请问是文件做过调整吗？

**Reply by LogCreative** [reply]
https://github.com/sjtug/SJTUThesis/blob/864c1551d049e0b84185c282df64e891be3a2056/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L48
